CONTRIBUTING.md

```diff
@@ -383,6 +383,87 @@ tested it becomes an officially supported Akka feature.
 
 [List of Akka features marked as may change](http://doc.akka.io/docs/akka/current/common/may-change.html)
 
+## Java APIs in Akka
+
+Akka, aims to keep 100% feature parity between the various language DSLs. Implementing even the API for Java in 
```

Comment:
"the various language" sounds like we many, "feature parity between Java and Scala APIs" ?